cmake/InstallConfigurations.cmake

```diff
@@ -0,0 +1,40 @@
+if (WIN32 AND SHARED)
+    install(
+            TARGETS raylib
+            ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
+            PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
+    )
+else ()
+    install(
+            TARGETS raylib
+            LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
+    )
+endif ()
```

Comment:
```suggestion
  install(
          TARGETS raylib EXPORT raylib-targets
          ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
          LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
          RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
          PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
  )
```

Reply:
@Neumann-A As it was configured before I extracted it into a separate file. I was planning on doing a few more passes after this refactor to clean up those stuff but thanks for the change I will take it.